src/site/content/en/fast/chrome-ux-report-data-studio-dashboard/index.md

```diff
@@ -13,137 +14,215 @@ date: 2018-11-05
+desktop and phone users. By default, the most recent month at the time you 
+created the dashboard is selected. To change between older or newer monthly 
+releases, use the "Month" filter at the top of the page.
 
-### Device distribution
+Note that tablet is omitted from these charts by default, but if needed you 
+could remove the "No Tablet" filter in the bar chart configuration, shown 
+below.
 
-The device distribution chart shows you the breakdown of phone, desktop, and
-tablet users over time. Many origins tend to have little to no tablet data so
-you'll see "0%" hanging off the edge of the chart.
+<img class="w-screenshot" src="./dash_cwv_tablet.png" alt="Editing the CrUX Dashboard to show tablets on the Core Web Vitals page">
 
-![CrUX Data Studio device dashboard](./dash_device.png)
+### Metric performance
 
-### Connection distribution
+After the Core Web Vitals page, you'll find standalone pages for all
+[metrics](https://developers.google.com/web/tools/chrome-user-experience-report/#metrics)
+in the CrUX dataset.
 
-The third and final page shows you the breakdown of effective connection types
-per month. There are five possible connection types (4G, 3G, 2G, Slow 2G, and
-Offline), but you will typically only see two or three in the chart. In the
-example above, 2G, Slow 2G, and offline are all 0%.
+<img class="w-screenshot" src="./dash_lcp.png" alt="CrUX Dashboard LCP page">
 
-![CrUX Data Studio effective connection type dashboard](./dash_ect.png)
+Atop each page is the "Device" filter, which you can use to restrict the form 
+factors included in the experience data. For example, you can drill down 
+specifically into phone experiences. This setting persists across pages.
 
-## Sharing the dashboard
+The primary visualizations on these pages are the monthly distributions of
+experiences categorized as "Good", "Needs Improvement", and "Poor". The 
+color-coded legend below the chart indicates the range of experiences included
+in the category. For example, in the screenshot above, you can see the 
+percent of good [Largest Contentful Paint](/lcp/) (LCP) experiences, which are 
+categorized as those less than 2500ms, fluctuating and getting slightly worse 
+in recent months.
 
-Data Studio dashboards can be shared just like a Google Doc to provide access to
-your team or to make the dashboard publicly viewable by permalink. Needless to
-say, you don't have to own the site you are getting a dashboard for, so you can 
-easily use this tool to keep an eye on your competitors' sites.
+The most recent month's percentages of good and poor experiences are shown 
+above the chart along with an indicator of the percent difference from the 
+previous month. For this origin, good LCP experiences fell by 3.2% to 56.04% 
+month-over-month.
 
-![CrUX Data Studio embedding and sharing](./dash_embed.png)
+{% Aside 'caution' %}
+Due to a quirk with Data Studio, you may sometimes see "No Data" here. This is
```

Comment:
```suggestion
Due to a quirk with Data Studio, you may sometimes see `No Data` here. This is
```
I'm not sure if this one is captured in the Style Guide but my convention is to use code formatting in these cases because it's literal text that you see on the screen